.github/workflows/e2e-nvidia-t4-x1.yml

```diff
@@ -88,13 +88,15 @@ jobs:
           nvidia-smi
           python3.11 -m pip cache remove llama_cpp_python
 
-          CMAKE_ARGS="-DLLAMA_CUDA=on" python3.11 -m pip install .
+          CMAKE_ARGS="-DLLAMA_CUDA=on -DLLAMA_NATIVE=off" python3.11 -m pip install .
 
           # https://github.com/instructlab/instructlab/issues/1821
           # install with Torch and build dependencies installed
           python3.11 -m pip install packaging wheel setuptools-scm
           python3.11 -m pip install .[cuda]
 
+          python3.11 -m pip install vllm@git+https://github.com/opendatahub-io/vllm@2024.08.01
```

Comment:
A couple of issues here:

1. Ideally this refer to vllm-requirements.txt instead of the direct reference so we have one place to bump versions/tags
2. These two install operations are not guaranteed to be compatible since each only has their respective dependency set. They must either be installed in a combined single install command, or they have to share a common constraints file.

I recommend replacing this section of install commands with the approach in the other workflows: https://github.com/instructlab/instructlab/blob/9e1770a86b5519c84f7b4d3d8bf12f2b3bd2d690/.github/workflows/e2e-nvidia-a10g-x4.yml#L124